package.json

```diff
@@ -12,7 +12,7 @@
     "check": "lerna run check --stream",
     "check-evergreen": "lerna run check",
     "bootstrap": "npm install && lerna run prepare --stream",
-    "bootstrap-evergreen": "npm ci && lerna run prepare",
+    "bootstrap-evergreen": "npm ci --registry https://registry.npmjs.org/ && lerna run prepare",
```

Comment:
I think this should not be required, initial bootstrap works fine on all machines in evergreen which means that npmrc in monorepo root is picked up by npm correctly, it's only `hadron-build` that fails during publishing because there is no `.npmrc` that npm can pick up inside this copied over version of the app.